service/systemd/service.go

```diff
@@ -77,7 +77,7 @@ func NewService(name string, conf common.Conf, dataDir string, newDBus DBusAPIFa
 	if conf.ExecStart != "" {
 		volName = renderer.VolumeName(common.Unquote(strings.Fields(conf.ExecStart)[0]))
 	}
-	dirName := volName + renderer.Join(dataDir, "init", name)
+	dirName := volName + renderer.Join(dataDir, "juju-init", name)
```

Comment:
This just needs to be "juju". The reason it was "init" previously was because it was under a generic /var/lib/juju location and there "init" was used to indicate what the dir was for. Here, we are already in a dir dedicated to systemd files. It may be worth looking into whether we even need a "juju" sub directory at all. It may be that all services are expected to just be placed together in the one directory. Can you look into what the systemd expectations / standard are in this area?